docs/jwst/tweakreg/README.rst

```diff
@@ -101,14 +101,14 @@ The ``tweakreg`` step has the following optional arguments:
   for matching. (Default=15)
 
 * ``searchrad``: A `float` indicating the search radius in arcsec for a match.
-  (Default=1.0)
+  (Default=2.0)
```

Comment:
I guess we'll have to wait till we've had more experience with real on-sky JWST observations using fine guide mode to have a better feel for the accuracy of blind pointing. I think everyone expects it to be better than 1 arcsec. I'm worried we may be driven right now by somewhat spurious and atypical results coming out of some commissioning data that were obtained in off-nominal circumstances (e.g. coarse guiding mode, etc.).